e2e/cypress/integration/ui/common/global_filter.spec.ts

```diff
@@ -1,28 +1,30 @@
 import { Project } from '../../../support/types';
 
 describe('global projects filter', () => {
+  const cypressPrefix = 'cypress-global-filter';
+  const now = Cypress.moment().format('MMDDYYhhmm');
+
   const proj1: Project = {
-    id: `cypress-project-1-${Cypress.moment().format('MMDDYYhhmm')}`,
+    id: `${cypressPrefix}-proj1-${now}`,
     name: 'Cypress Project 1',
     skip_policies: true
   };
   const proj2: Project = {
-    id: `cypress-project-2-${Cypress.moment().format('MMDDYYhhmm')}`,
+    id: `${cypressPrefix}-proj2-${now}`,
     name: 'Cypress Project 2',
     skip_policies: true
   };
   const proj3: Project = {
-    id: `cypress-project-3-${Cypress.moment().format('MMDDYYhhmm')}`,
+    id: `${cypressPrefix}-proj3-${now}`,
     name: 'Cypress Project 3',
     skip_policies: true
    };
-  const pol_id = `cypress-policy-${Cypress.moment().format('MMDDYYhhmm')}`;
-  const nonAdminUsername = `nonadmin-${Cypress.moment().format('MMDDYYhhmm')}`;
+  const pol_id = `${cypressPrefix}-pol-${now}`;
```

Comment:
[nit] it's a bit uncommon to use snakecase, I believe
```suggestion
  const polId = `${cypressPrefix}-pol-${now}`;
```